tests/wdio/loaders-spec.js

```diff
@@ -0,0 +1,47 @@
+Terra.describeViewports('loaders', ['huge'], () => {
+  it('loads a js codeblock', () => {
+    browser.url('/raw/test/terra-dev-site/loaders/codeblock-js');
+    Terra.validates.element('js codeblock');
+  });
+
+  it('loads a json codeblock', () => {
+    browser.url('/raw/test/terra-dev-site/loaders/codeblock-json');
+    Terra.validates.element('json codeblock');
+  });
+
+  it('loads a ts codeblock', () => {
+    browser.url('/raw/test/terra-dev-site/loaders/codeblock-ts');
+    Terra.validates.element('ts codeblock');
+  });
+
+  it('loads a package', () => {
+    browser.url('/raw/test/terra-dev-site/loaders/package');
+    Terra.validates.element('package');
+  });
+
+  it('loads a props table', () => {
+    const viewport = browser.getViewportSize();
+    viewport.height = 3000;
+    browser.setViewportSize(viewport);
+    browser.url('/raw/test/terra-dev-site/loaders/props-table');
+    Terra.validates.element('props table');
+  });
+
+  it('loads an example', () => {
+    const viewport = browser.getViewportSize();
+    viewport.height = 3000;
+    browser.setViewportSize(viewport);
```

Comment:
do we need to change the viewport height? If so, we should be resetting it after the screenshot

Reply:
yes the viewport hight needs to be higher, but it also needs to be higher for the subsequent tests.